capstone/home.html

```diff
@@ -0,0 +1,134 @@
+      <a class="menu-button close" href="#"><i class="fa-solid fa-xmark"></i></a>
+      <div hidden class="menu-content">
+        <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+        <div class="menu-items">
+          <a class="aboutPage" href="about.html">About</a>
+          <a href="home.html#program">Program</a>
+          <a href="home.html#join">Join</a>
+          <a href="#sponsor">Sponsor</a>
+          <a href="about.html#news">News</a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="main">
+      <div class="homeContent home">
+        <h2 class="greet">"Hello! Rock & Roll Fans"</h2>
+        <h1 class="title">ROCK & ROLL HALL OF FAME 2023</h1>
+        <p class="description">The ROCK & ROLL Hall of Fame celebrates the sound of youth culture and honors the artists whose music connects us all. Join us in our next presentations in April, Cleveland.</p>
+        <h2 class="date">2023.04.27(THU) - 29(SAT)</h2>
+        <h3 class="location">@ The Rock and Roll Hall of Fame Museum, Cleveland</h3>
+      </div>
+    </div>
+    <div id="program" class="mainProgram">
+      <div class="holder">
+        <h2 class="title">Main Program</h2>
+        <div class="underLine"></div>
+      </div>
+      <div class="programElements">
+        <div class="elements">
+          <i class="fa-sharp fa-solid fa-hourglass-start"></i>
+          <h3>Tour starts</h3>
+          <p>It's a long way to the top if you want to rock & roll, sometimes you have to start at the bottom. </p>
+        </div>
+        <div class="elements">
+          <i class="fa-solid fa-guitar"></i>
+          <h3>The Garage</h3>
+          <p>We're getting the band back together! The Garage is fully opened and ready to rock.</p>
+        </div>
+        <div class="elements">
+          <i class="fa-solid fa-star"></i>
+          <h3>Hall of Fame</h3>
+          <p>Inductee Signature Gallery, New Inductee exhibit. Featuring the Power of Rock Experience.</p>
+        </div>
+        <div class="elements">
+          <i class="fa-solid fa-map"></i>
+          <h3>THE WALL</h3>
+          <p>Stand beneath the Pink Floyd tour replica of The Wall, Berlin performance July 21, 1990.</p>
+        </div>
+        <div class="elements">
+          <i class="fa-solid fa-ranking-star"></i>
+          <h3>Legends of Rock</h3>
+          <p>A myriad of musical genres and highlights some of the world’s most iconic acts both past and present.</p>
+        </div>
+      </div>
+      <a href="#" class="programDetails">SEE THE WHOLE PROGRAM</a>
+      <button id="join" class="programBtn">Join ROCK & ROLL Hall of Fame Tour 2023</button>
+    </div>
+    <section id="artistSection">
+      <div id="featuredArtists"></div>
+    </section>
+  </main>
+  <footer class="more">
+    <div id="sponsor" class="partners">
+      <div class="holder">
+        <h2 class="title">Partner</h2>
```

Comment:
- [ ] You did well here, but kindly try to stick to the Behance, so please try to use the correct color of the Behance here😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/217925364-dfc3d0ce-29b0-43b1-a069-8408a49180a0.png)   |    ![image](https://user-images.githubusercontent.com/83360636/217925333-cfc29d7e-4ef2-4fba-9fd0-4fd8d8b92997.png) |